### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -279,6 +279,15 @@ websites:
       doc: https://semaphoreci.com/docs/two-step-verification.html
       software: Yes
 
+    - name: Sentry
+      url: https://getsentry.com
+      img: sentry.png
+      tfa: Yes
+      sms: Yes
+      hardware: Yes
+      software: Yes
```
Comment (Carlgo11) [inline _data/developer.yml:288]: I can't find anything about software or SMS on that doc page.

Comment (stephengroat) [reply _data/developer.yml:288]: In the blog post, first line:
    We think security is important so we’ve rolled out two-factor authentication for Sentry. You can use U2F devices (like yubikeys), Google Authenticator (or similar apps supporting TOTP), or SMS as second factor now.

Comment (Carlgo11) [reply _data/developer.yml:288]: oh! 😄

Comment (stephengroat) [reply _data/developer.yml:288]: I also signed up for an account and confirmed all 3 options are valid

**Pull request merged**